### ISSUE TITLE
Current ipfs install has numerous dependency warnings.

### ISSUE THREAD

**Issue by mitra42** [ipfs/js-ipfs#2347]
- **Version**: 0.35.0
- **Platform**: 
Linux box.lan 4.19.58-v7+ #1245 SMP Fri Jul 12 17:25:511 BST 2019 armv71 GNU/Linux
and replicated on ...
Darwin mitraglass-233160.local 18.7.0 Darwin Kernel Version 18.7.0: Thu Jun 20 18:42:21 PDT 2019; root:xnu-4903.270.47~4/RELEASE_X86_64 x86_64
- **Subsystem**: Installation

#### Description: 
IPFS has warnings on installation of three moved packages (boom, hoi and joi)
```
warning @internetarchive/dweb-objects > @internetarchive/dweb-transports > ipfs > boom@7.3.0: This module has moved and is now available at @hapi/boom. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning @internetarchive/dweb-objects > @internetarchive/dweb-transports > ipfs > hapi@18.1.0: This module has moved and is now available at @hapi/hapi. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues.
warning @internetarchive/dweb-objects > @internetarchive/dweb-transports > ipfs > joi@14.3.1: This module has moved and is now available at @hapi/joi. Please update your dependencies as this version is no longer maintained an may contain bugs and security issues
```
Note the warning about hapi is also in ipfs-mfs  and libp2p-webrtc-star and ipfs-unixfs-importer  (but I'm assuming you don't want a duplicate of this issue there ?)
And that the outdated hapi generates a bunch more warnings about its dependencies which are of course also outdated. 

Note ... I'm reporting it, because packages including the Internet Archive dweb-transports package (which depends on IPFS)  are (not unreasonably) complaining to me :-(

#### Steps to reproduce the error:
Run `yarn upgrade` on existing node install

**Comment by alanshaw** [comment]
@mitra42 you'll need to upgrade to `ipfs@0.37` to resolve these errors.

js-ipfs depends on the moved versions now:

https://github.com/ipfs/js-ipfs/blob/3cade672b1c3d2aa19394e2980cfad9cd94764dc/package.json#L62-L64

**Comment by mitra42** [comment]
Great - only seeing two dependency warnings now from IPFS, both of which are probably out of your control 
```
warning @internetarchive/dweb-objects > @internetarchive/dweb-transports > ipfs > @hapi/joi > @hapi/hoek@6.2.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
warning @internetarchive/dweb-objects > @internetarchive/dweb-transports > ipfs > hapi-pino > @hapi/hoek@6.2.4: This version has been deprecated in accordance with the hapi support policy (hapi.im/support). Please upgrade to the latest version to get the best features, bug fixes, and security patches. If you are unable to upgrade at this time, paid support is available for older versions (hapi.im/commercial).
```
I'll raise an issue on those repo's

**Comment by mitra42** [comment]
hapi-pino Reported at https://github.com/pinojs/hapi-pino/issues/82 and hapi/joi looks like it willl fix itself as joi 16.0.1 points at a later version of hoek but is still itself in preview
